### ISSUE TITLE
Switch CLI to a non-system language

### ISSUE THREAD

**Issue by TomasHubelbauer** [dotnet/cli#7585]
How do I make the `dotnet` CLI be in English when I'm on non-English Windows?

**Comment by johnbeisner** [comment]
I have changed the language from English to something else on an English OS - however, it's probably the same technique:
In your command shell; set the environmental variable "DOTNET_CLI_UI_LANGUAGE" to "en"

Let me know if that works - I'm now curious...

**Comment by TomasHubelbauer** [comment]
Thank you @johnbeisner, `setx DOTNET_CLI_UI_LANGUAGE en` seems to have had the desired effect.

FWIW I think this is a bad default and changing language away from English to the system language should be opt-in (dismissable hint to do it is fine though).

**Comment by TomasHubelbauer** [comment]
Sorry, this did not end up working out after all. I am not receiving `CSC : warning CS2038: Language code cs in invalid`.

In my `<project>.AssemblyInfo.cs` file I have Czech generated comments. :-( The CLI is in English though…

Really hope there is a straightforward way to just switch everything to English and ignore the system language, because I sometimes have to work on a machine with Home edition which won't allow me to switch languages and this is extremely frustrating.

**Comment by TomasHubelbauer** [comment]
Might this help when adjusted for .NET Core paths?

[Change compiler errors language in Visual Studio 2013](https://stackoverflow.com/a/22529794/2715716)

**Comment by TomasHubelbauer** [comment]
And it stayed that way for exactly one compiler invocation. Deleted `1029` afterwards to no avail. It's in Czech again.

**Comment by livarcocc** [comment]
cc @nguerrera

**Comment by nguerrera** [comment]
@TomasHubelbauer How are you invoking the compilation: `msbuild` or `dotnet build`?

**Comment by TomasHubelbauer** [comment]
@nguerrera `dotnet watch run`

**Comment by livarcocc** [comment]
Does this repro when you just do dotnet build?

**Comment by TomasHubelbauer** [comment]
@livarcocc I have just noticed `DOTNET_CLI_UI_LANGUAGE` remained set to `cz`, I must have set it only for the PowerShell session when I tried changing it the last time. Aplogies.

This, presumably combined with the fact that I have deleted the resource directories, now made the compiler spit out warning `CS2038` about `cz` language code not being valid (configured `cz` in `DOTNET_CLI_UI_LANGUAGE` but the `1029` [Czech] directory is missing). The warning is in Czech (?!) but the exception messages thrown during the execution of the program are in English.

I have set the variable using `[Environment]::SetEnvironmentVariable("DOTNET_CLI_UI_LANGUAGE", "en", "User")` now as opposed to `$Env:DOTNET_CLI_UI_LANGUAGE=en` I must have used previously, logged out and back in and it finally works and I have everything in English. 💯 

Thank you!
